model.go

```diff
@@ -524,6 +524,7 @@ func (m *Model) updateActivePane(msg tea.Msg) tea.Cmd {
 		m.ContentStyle = DefaultStyles(m.config).Content.Blurred
 		m.FoldersStyle = DefaultStyles(m.config).Folders.Focused
 		m.Folders, cmd = m.Folders.Update(msg)
+		m.saveConfig()
```

Comment:
Is it ok to write synchronously here or should we do it within a `tea.Cmd`?

Reply:
That sounds like a reasonable decision.  Updated in c7b28441af3acea6d82f89b5c0f09c15d41a4cf9.